packages/web-components/src/utils/helpers.ts

```diff
@@ -692,3 +692,12 @@ export const checkSlotInChildMutations = (
 
 export const isElInAGGrid = (el: HTMLElement): boolean =>
   !!el.closest(".ag-cell") && !!el.closest(".ag-root");
+
+/*
+ * Checks if the component is slotted in its relevant 'group' component
+ * @param component - the component to check
+ */
+export const isSlottedInGroup = (component: any): boolean => {
```

Comment:
Couldn't `this.el` be passed to the function? Doesn't seem like the whole object.